src/buildHTML.js

```diff
@@ -1491,6 +1491,22 @@ groupTypes.mclass = function(group, options) {
     return makeSpan([group.value.mclass], elements, options);
 };
 
+groupTypes.transform = function(group, options) {
+    const span = groupTypes.text(group, options);
+    if (group.value.dy) {
+        const dy = group.value.dy.value;
+        span.style.top = "" + (-dy.number) + dy.unit;
+        // Not working yet:
+        if (dy.number > 0) {
+            span.height += calculateSize(dy.number, options.style);
+        } else {
+            span.depth -= calculateSize(dy.number, options.style);
+        }
```

Comment:
> At least, my mental model is that height is the extent above the baseline (y-max) while depth is the extent below the baseline (y-min) -- please correct me if I'm wrong.

That's right, but if a glyph is shifted vertically, the the extent below the baseline will be different.

What happens if you put the test expression inside a `\sqrt` or a `\frac`?  It may be that just our wrapper container isn't doing the right thing with the `height` and `depth` you're setting on the `span`.